jar-filter/build.gradle

```diff
@@ -22,6 +22,21 @@ gradlePlugin {
     automatedPublishing = false
 }
 
+pluginBundle {
+    website = project.vcsUrl
+    vcsUrl = project.vcsUrl
+    description = 'Corda JAR Filter Plugin'
+    tags = ['corda']
+
+    plugins {
+        jarFilterPlugin {
+            displayName = 'Corda JAR Filter Plugin'
+        }
+    }
+}
+
+task('publishPlugin').dependsOn jar
```

Comment:
I think you forgot to remove the dummy `publishPlugin` task declaration from `jar-filter` :stuck_out_tongue:.